### PR TITLE
Add info command

### PR DESCRIPTION
**Pull request by bouthilx**
todo

### CODE REVIEW

File: src/orion/core/cli/info.py [hunk 1]
```diff
@@ -0,0 +1,524 @@
...
+
+
+def format_info(experiment, templates=None):
+    """Render a string for all info of experiment
+
+    Parameters
+    ----------
+    experiment: `orion.core.worker.experiment.Experiment`
+    templates: dict
+        templates for all sections and titles
```
Comment (DatCorno) [inline src/orion/core/cli/info.py]: Uppercase.

File: src/orion/core/cli/info.py [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+#             print_dict(item, depth + 1, width=width)
+#         else:
+#             print(" " * (depth * width), "- ", item, sep='')
+
+
+# def print_experiment_tree(node, offset=0, width=2):
+#
+#     print(" " * (offset * width), node.item, sep="")
+#     for child in node.children:
+#         print_experiment_tree(child, offset=offset + 1, width=width)
```
Comment (DatCorno) [inline src/orion/core/cli/info.py]: Delete this?

**Comment by bouthilx** [comment]
[add `orion info`](https://trello.com/c/DGGNpdPq/25-add-orion-info)

File: src/orion/core/cli/info.py [hunk 3]
```diff
@@ -0,0 +1,508 @@
...
+
+{metadata}
+
+{refers}
+
+{stats}
+"""
+
+
+def format_info(experiment, templates=None):
```
Comment (DatCorno) [inline src/orion/core/cli/info.py]: Is the `templates` keyword argument for future configuration?

Comment (bouthilx) [reply src/orion/core/cli/info.py]: It's to support different formatting, but to be honest it is probably a terrible overkill...

File: src/orion/core/cli/info.py [hunk 4]
```diff
@@ -0,0 +1,508 @@
...
+      - 4
+
+        - 5
+        - 6
+        - 7
+        - 8
+
+    """
+    # To avoid using mutable objects as default values in function signature.
+    if templates is None:
```
Comment (DatCorno) [inline src/orion/core/cli/info.py]: Can't we move all these `if`s to the `format_info` function?

File: tests/unittests/core/cli/test_info.py [hunk 5]
```diff
@@ -0,0 +1,949 @@
...
+            2.1: "2.1.1",
+            2.2: {}
+        },
+        3: {}
+    }
+
+
+@pytest.fixture
+def dummy_list_of_lists():
+    """Return an list of lists"""
```
Comment (DatCorno) [inline tests/unittests/core/cli/test_info.py]: Typo, `a`

File: tests/unittests/core/cli/test_info.py [hunk 6]
```diff
@@ -0,0 +1,949 @@
...
+        1,
+        [2, 3],
+        4,
+        [5, 6, 7, 8]
+    ]
+
+
+@pytest.fixture
+def dummy_list_of_objects(dummy_dict):
+    """Return an list of objects"""
```
Comment (DatCorno) [inline tests/unittests/core/cli/test_info.py]: Typo, `a`.

File: tests/unittests/core/cli/test_info.py [hunk 7]
```diff
@@ -0,0 +1,949 @@
...
+def algorithm_dict():
+    """Return an algorithm configuration"""
+    return dict(
+        bayesianoptimizer=dict(
+            acq_func='gp_hedge',
+            alpha=1e-10,
+            n_initial_points=10,
+            n_restarts_optimizer=0,
+            normalize_y=False,
+            strategy='cl_min'))
```
Comment (DatCorno) [inline tests/unittests/core/cli/test_info.py]: We don't support `strategy` anymore, should we remove this?

File: src/orion/core/cli/info.py [hunk 8]
```diff
@@ -0,0 +1,397 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.core.cli.info` -- Module to info experiments
+========================================================
+
+.. module:: info
+   :platform: Unix
+   :synopsis: List experiments in termnial
```
Comment (DatCorno) [inline src/orion/core/cli/info.py]: What is this ;)?

File: tests/unittests/core/cli/test_info.py (hunk 7 above)
Comment (bouthilx) [reply tests/unittests/core/cli/test_info.py]: Indeed!

**Pull request merged**